samples/bot-conversation-sso-quickstart/js/.gitignore

```diff
@@ -26,7 +25,6 @@ yarn-error.log*
 
 # TeamsFx files
 env/.env.*.user
-env/.env.local
```

Comment:
Revert it back. 
Just commit the `.env.local` file, so that the user could refer to it.
But for the user, `.env.local` file would not be committed to source control by default.

Reply:
reverted changes for gitignore files